bazel/test_for_benchmark_wrapper.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Set the benchmark time to 0 to just verify that the benchmark runs to completion.
+"${TEST_SRCDIR}/envoy/$@" --benchmark_min_time=0
```

Comment:
Is the only purpose of this to feed in the test arg? `cc_test` takes an `args` parameter that could also do that if it's the case. If there's some other use, then this is fine.

Reply:
The test involves just running the benchmark cc_binary.  There's no true test.cc